.github/workflows/enforce-labels.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     steps:
     - uses: yogevbd/enforce-label-action@2.1.0
       with:
-        REQUIRED_LABELS_ANY: "verified,trivial,needs-verification"
-        REQUIRED_LABELS_ANY_DESCRIPTION: "PR must be labeled with 'trivial', 'needs-verification', or 'verified'"
-        BANNED_LABELS: "do-not-merge,duplicate,invalid,wontfix"
+        REQUIRED_LABELS_ANY: "verified,trivial,needs-verification,zero impact"
```

Comment:
Do we still need trivial? I feel like that's superseded by zero impact.

Reply:
@Blainegunn where do we stand on this

Reply:
If they have the same meaning, why not continue with just `trivial`, since contributors are already familiar with that one?

Reply:
With the new flow I'd say there is:
- zero-impact: this won't or can't have any production impact (Unit tests, Github Actions, etc.)
- needs-verification/verified: a change has been tested and is ready for stage (Any change that touches anything visitor facing)

I'd vote for removing trivial, as the stage process doesn't have a concept of small or big PRs and every visitor facing change should be tested IMO

Reply:
As discussed:
`zero-impact` can always be included in any stage batch
`trivial` small adjustments should be uncomplicated and quick to get merged to main, without big concerns to break prod//have bugs slip through the SOTs testing, but still get some attention
`needs-verification/verified` bigger PRs